lib/masamune/transform/postgres/snapshot_dimension.psql.erb

```diff
@@ -41,15 +41,15 @@ WITH ranges AS (
     <%- end -%>
     consolidated.start_at
   FROM (
-    SELECT DISTINCT ON (<%= target.window('start_at', 'id').join(', ') %>)
```

Comment:
not sure why `id` is in this `SELECT DISTINCT` clause- seems vestigial and defeats the purpose